app/views/guidance/check_data.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "guidance/check_data" %>
```

Comment:
We can delete this file if we pass the `format` to the render call in the controller.